jiant/tasks/tasks.py

```diff
@@ -198,6 +198,8 @@ class Task(object):
         - optimizer
     """
 
+    CSV_PREDS = False
```

Comment:
Add a comment explaining this.

Reply:
Removed, and replaced with a direct check for task class.